test/cpp/api/jit.cpp

```diff
@@ -49,12 +49,12 @@ TEST(TorchScriptTest, TestNestedIValueModuleArgMatching) {
   list_of_lists.push_back(list);
   module->run_method("nested_loop", list_of_lists, b);
 
-  c10::impl::GenericList generic_list;
-  c10::impl::GenericList empty_generic_list;
+  auto generic_list = c10::impl::GenericList(c10::impl::deprecatedUntypedList());
```

Comment:
maybe fix this test to not use deprecated stuff?

Reply:
yes, there's some call sites that can be easily fixed. I didn't do it yet because it's more important to land the API change on time for the 1.2 release, we can clean these up later.